From bb89a63eed58c986ef0de7c3cef21225542dcdc0 Mon Sep 17 00:00:00 2001
From: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Date: Fri, 23 Feb 2024 09:16:14 +0100
Subject: [PATCH 0650/1695] dts: stm32: activate HSI calibration on STM32MP23
 boards

Default enable HSI (High Speed Internal oscillator) calibration. This
internal oscillator tends to derive over time so calibration is needed.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 2b8558d9d30a624de8ede8115b454d0b5966f1b2 dts: stm32: activate HSI calibration on STM32MP23 boards]
Signed-off-by: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Change-Id: Ib980ddd68dee33a89a8097e343a5638b20baf74b
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/362470
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 core/arch/arm/dts/stm32mp235f-dk.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/arch/arm/dts/stm32mp235f-dk.dts b/core/arch/arm/dts/stm32mp235f-dk.dts
index cc97cbf28..9bc1d73f8 100644
--- a/core/arch/arm/dts/stm32mp235f-dk.dts
+++ b/core/arch/arm/dts/stm32mp235f-dk.dts
@@ -53,6 +53,10 @@
 	cpu-supply = <&vddcpu>;
 };
 
+&hsi_calibration {
+	status = "okay";
+};
+
 &i2c7 {
 	pinctrl-names = "default", "sleep";
 	pinctrl-0 = <&i2c7_pins_a>;
-- 
2.47.2

